Tests/ExperimentalTableDrivenSwiftProtobufTests/unittest.pb.swift

```diff
@@ -4637,7 +4684,7 @@ extension SwiftProtoTesting_TestRequiredMessage: SwiftProtobuf.Message, SwiftPro
   }
 
   static func ==(lhs: SwiftProtoTesting_TestRequiredMessage, rhs: SwiftProtoTesting_TestRequiredMessage) -> Bool {
-    fatalError("table-driven == not yet implemented")
+    return lhs._storage.isEqual(to: rhs._storage)
```

Comment:
Actually, can this move fully into the runtime or do we need to generate it into every message since it's now the same?

Reply:
ack.